examples/SMIRNOFF_comparison/compare_molecule_energies.py

```diff
@@ -1,7 +1,7 @@
 #!/bin/env python
 
 from openforcefield.utils import *
-from openforcefield.typing.engines.smirnoff import ForceField
+from openforcefield.typing.engines.smirnoff import * 
```

Comment:
We should avoid `import *` if at all possible, since this practice is [discouraged by PEP 8](https://www.python.org/dev/peps/pep-0008/#id23) (though we have a lot of non-PEP8-compliant code because we modeled this off of OpenMM, which uses C++ conventions). For now, it's fine to get things working though.